winlocalprocessspawner/win_utils.py

```diff
@@ -12,6 +17,76 @@
 CLOSEHANDLE.argtypes = [HANDLE]
 CLOSEHANDLE.restype = BOOL
 
+GENERIC_ACCESS = win32con.GENERIC_READ | win32con.GENERIC_WRITE | win32con.GENERIC_EXECUTE | \
+    win32con.GENERIC_ALL
+
+WINSTA_ALL = (win32con.WINSTA_ACCESSCLIPBOARD | win32con.WINSTA_ACCESSGLOBALATOMS |
```

Comment:
some of this identations are off. Could you run pylint to make sure the new conforms to it? Mainly looking at identations and variable names  warning on the new code. I have a pending task to get a CI running that will do this checks automatically, hope I can get to it soon....

Reply:
Done